src/ngraph/runtime/gpu/gpu_emitter.cpp

```diff
@@ -996,6 +996,45 @@ CUDNN_SAFE_CALL(cudnnSetOpTensorDescriptor(opTensorDesc,
                 writer.block_end();
             }
 
+            template <>
+            void GPU_Emitter::EMITTER_DECL(ngraph::op::FunctionCall)
+            {
+                auto function_call = static_cast<const ngraph::op::FunctionCall*>(node);
+                shared_ptr<Function> function = function_call->get_functions()[0];
```

Comment:
Why only the first function?

Reply:
check the spec, it's actually a single function even it use vector. @bob correct me if I'm wrong.

Reply:
https://github.com/NervanaSystems/ngraph/blob/3d66cba498b0c8076a82b52d29a4d25a691698f5/src/ngraph/op/function_call.cpp#L63

Reply:
Ah I see, an overriding function that in this case it returns a singleton vector.

nit: could consider using `get_functions().back()` to suggest it's a unit length vector